### PR TITLE
setting for poster instead of thumbs - tempo dialog

### PR DESCRIPTION
**Pull request by DaVukovic**
This is a pretty big change and I had to squash some commits due to merge errors I had in my fork

This PR adds PR #117 which was introduced by @gade01 . The branch needed a rebase, but the remote branch was already deleted. Hence I've added the changes manually

This PR also contains changes requested by #198 and introduces a setting which optionally shows poster instead of thumbs if available. 

I tried to test everything I could imaging. Hopefully I haven't forgotten anything. 

Feel free to tell me if anything needs to be changed.

### CODE REVIEW

File: 720p/DialogSeekBar.xml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <window>
 	<defaultcontrol>1</defaultcontrol>
-	<visible>Player.Seeking | Player.HasPerformedSeek(3) | Player.Paused | Player.Forwarding | Player.Rewinding | Player.ShowTime | Player.IsTempo</visible>
+	<visible>Player.Seeking | Player.DisplayAfterSeek | Player.Paused | Player.Forwarding | Player.Rewinding | Player.ShowTime | [Player.IsTempo + !System.IdleTime(3)]</visible>
```
Comment (bossanova808) [inline 720p/DialogSeekBar.xml:4]: I think you need to rebase this PR on top of my PR committed last few days, as this reverts some of those changes (should be `Player.HasPerformedSeek(3)` not `Player.DisplayAfterSeek`)

I'd guess there are likely other missed changes that would be reverted by this

**Comment by gade01** [comment]
@DaVukovic Thanks for updating this.
I will look at it as soon as possible.

It's true that we need to use `Player.HasPerformedSeek(3)` instead of `Player.DisplayAfterSeek` in Kodi 20.

**Comment by DaVukovic** [comment]
Damn...thanks

I´ve got some merge problems and I thought I did it all well. 

Will have a look. Would be nice if you could point out other flaws if any. 

Thanks much :+1:

**Comment by gade01** [comment]
Everything else looks great ;)

**Comment by bossanova808** [comment]
I have a work deadline today so just did a quick scan - just a couple of instances of that seek issue to be fixed but I can't on first glance see any other issues with the last PR.

Thanks for your work!  New features in Confluence in 2023 is pretty cool!

**Comment by DaVukovic** [comment]
Fix is pushed.

**Comment by gade01** [comment]
@DaVukovic There is still 3 `Player.DisplayAfterSeek` that should be replaced by `Player.HasPerformedSeek(3)`

**Comment by DaVukovic** [comment]
@gade01 Thanks for pointing me to it. 

Solved now

**Review by gade01 — APPROVED** [review]
@DaVukovic everything looks great now, thanks.

**Comment by DaVukovic** [comment]
ok. Cool. 

So let's get this startet. 

I'll merge this now and will then create a new branch for Nexus to submit it to the Kodi repo

**Pull request merged**